rfc/organizing-work.md

```diff
@@ -0,0 +1,58 @@
+# Organizing work
+
+![status:draft](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square)
+
+## Editors
+
+- [Alan Shaw](https://github.com/alanshaw), [Protocol Labs](https://protocol.ai/)
+
+## Authors
+
+- [Alan Shaw](https://github.com/alanshaw), [Protocol Labs](https://protocol.ai/)
+
+## Abstract
+
+Formalize a process for working that ensures a consistent and up to date view of the work items in our pipeline. Serves to document a process that can be onboarded and observed by others.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Out of scope
+
+* The tools or locations used to document work items. i.e. Github issues, Jira, Notion etc.
+* Roadmapping/general high level direction setting.
+
+## Pipeline
+
+Generally a work item would move from inception to completion through a pipeline in the following order:
+
+1. **Heap** - anyone can pile stuff onto the heap. It's just a place to track things that we may want to work on in the future. It can be of arbitrary size and detail.
+
+    Items that are of ambiguous _certainty_ SHOULD be accompanied by an [RFC](https://github.com/web3-storage/rfc) - a PR that allows others to comment in an open and structured way on the item. For example, an RFC may be a new idea or a way of achieving a goal that could be done in multiple ways. An RFC allows others to validate the idea, question or assumption to determine with more certainty if it should be worked on.
+    
+    All members SHOULD read RFC documents and provide feedback to allow the author to determine validity of the work item and adjust or clean up if necessary.
```

Comment:
```suggestion
    All members MAY read RFC documents and provide feedback to allow the author to determine validity of the work item and adjust or clean up if necessary.
```

I would frame more as everyone can participate and have time to do so. But also can opt out from participation

Reply:
I agree with @vasco-santos participation is voluntary unless you're input was explicitly requested.